examples/webgl_camera.html

```diff
@@ -110,30 +110,13 @@
 
 				//
 
-				const geometry = new THREE.BufferGeometry();
-				const vertices = [];
-
-				for ( let i = 0; i < 10000; i ++ ) {
-
-					vertices.push( THREE.MathUtils.randFloatSpread( 2000 ) ); // x
-					vertices.push( THREE.MathUtils.randFloatSpread( 2000 ) ); // y
-					vertices.push( THREE.MathUtils.randFloatSpread( 2000 ) ); // z
-
-				}
-
-				geometry.setAttribute( 'position', new THREE.Float32BufferAttribute( vertices, 3 ) );
-
-				const particles = new THREE.Points( geometry, new THREE.PointsMaterial( { color: 0x888888 } ) );
-				scene.add( particles );
-
-				//
-
```

Comment:
I've thought the particles make the example a bit more interesting so i vote to keep them.

Reply:
OK